pom.xml

```diff
@@ -2682,6 +2685,26 @@
         <artifactId>ecj</artifactId>
         <version>${version.ecj}</version>
       </dependency>
+
+      <dependency>
```

Comment:
This is `<dependencyManagement>` so we should not be declaring scopes here (just defining versions). Scope should be defined at the level of individual poms that actually use these in their `<dependenciese>` section. Please remove the `<scope>runtime</scope>` from all three deps..

Reply:
Fixed was copy and paste